host.go

```diff
@@ -268,33 +268,36 @@ func (h *gpbftRunner) BroadcastMessage(msg *gpbft.GMessage) error {
 
 var _ pubsub.ValidatorEx = (*gpbftRunner)(nil).validatePubsubMessage
 
-func (h *gpbftRunner) validatePubsubMessage(ctx context.Context, pID peer.ID,
-	msg *pubsub.Message) pubsub.ValidationResult {
+func (h *gpbftRunner) validatePubsubMessage(_ context.Context, _ peer.ID, msg *pubsub.Message) (_result pubsub.ValidationResult) {
+	start := time.Now()
+	defer func() {
+		recordValidationTime(start, _result)
```

Comment:
Any reason not to feed the context through?

Reply:
Because I thought OTEL will not record the metric if the context passed to metric is cancelled.

Based on a quick local test that is not true, and it measures things regardless of the state of context passed to a metric, which is slightly confusing for me.

Thanks Steven